src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/BarnTilRevurdering.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+
+import { BodyShort, Checkbox, CheckboxGroup, List } from '@navikt/ds-react';
+
+import { useApp } from '../../../../context/AppContext';
+import DataViewer from '../../../../komponenter/DataViewer';
+import { Ressurs } from '../../../../typer/ressurs';
+
+export interface BarnTilRevurderingResponse {
+    barn: BarnTilRevurdering[];
+}
+
+interface BarnTilRevurdering {
+    ident: string;
+    navn: string;
+    finnesPåForrigeBehandling: boolean;
+}
+
+const filterEksisterendeBarn = (barnTilRevurdering: BarnTilRevurderingResponse) =>
+    barnTilRevurdering.barn.filter((b) => b.finnesPåForrigeBehandling);
+
+const filterValgbareBarn = (barnTilRevurdering: BarnTilRevurderingResponse) =>
+    barnTilRevurdering.barn.filter((b) => !b.finnesPåForrigeBehandling);
+
+const BarnTilRevurdering: React.FC<{
+    fagsakId: string;
+    barnTilRevurdering: Ressurs<BarnTilRevurderingResponse>;
+    settBarnTilRevurdering: React.Dispatch<
+        React.SetStateAction<Ressurs<BarnTilRevurderingResponse>>
+    >;
+    settValgteBarn: (identer: string[]) => void;
+}> = ({ fagsakId, barnTilRevurdering, settBarnTilRevurdering, settValgteBarn }) => {
+    const { request } = useApp();
+
+    const hentBarnTilRevurdering = (fagsakId: string) => {
+        request<BarnTilRevurderingResponse, null>(
+            `/api/sak/behandling/barn-til-revurdering/${fagsakId}`
+        ).then(settBarnTilRevurdering);
+    };
+
+    useEffect(() => {
+        hentBarnTilRevurdering(fagsakId);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
Burde fagsakId inn som enn dependency for å unngå feil i fremtiden? (Mulig hele komponenten rendres på nytt 🤔 )

Reply:
Evt. også flytte `hentBarnTilRevurdering` inn i useEffecten så man slipper å disable eslint 🤓 

Ref https://react.dev/reference/react/useEffect#removing-unnecessary-function-dependencies

Reply:
La til den. Hele komponenten vil bli rendert på nytt